### ISSUE TITLE
Toggle on climate entity, turns from toggle to text of state

### ISSUE THREAD

**Issue by Alfiegerner** [benct/lovelace-multiple-entity-row#18]
Hi,

I'm using this with toggles on climate entities. When they are 'off' the toggle appears, when they are on the text of the state appears, e.g. 'cool'. 

This wasn't happening before a recent update. At some point I think a HA upgrade broke the toggle functionality, then I upgraded the js file for this and now it's half working as described. Could well be a feature of the climate component maybe?

**Comment by Alfiegerner** [comment]
The problem is the validation that checks if the toggle is valid in this method:

```
    validateToggle(config, stateObj) {
        return config && config.toggle === true && stateObj && (stateObj.state === "on" || stateObj.state === "off");
    }
```
It turns out some domains like climate can be toggled, even if their state is neither on or off.  I'm not sure how many other domains might be affected by this, for now I've just locally modified this method to return true.  Maybe condition on domain type? Lights, switch, lock, climate ..... not sure.

**Comment by benct** [comment]
Hi, sorry for the late reply.

I have been using the same code for checking the toggles as the official HA frontend, so that's a bit of a surprise. But as you say, I might have missed a condition on some domain types or something. I think it works for most domains as it is, but I will have a closer look at trying to fix this when I get the time.

**Comment by benct** [comment]
Should be fixed in version `2.0.0`. Let me know if not :)
